src/mixins/uiMention.js

```diff
@@ -24,6 +25,13 @@ export default {
 					}
 				}
 
+				// Workaround to add the current user to the suggestions if they are not already in the list+
+				// https://github.com/nextcloud/server/issues/48180
+				const currentUser = getCurrentUser()
+				if (!users.some(user => user.id === currentUser.uid) && currentUser.uid.includes(text)) {
```

Comment:
We should also check the displayname so you can find yourself in the matches, can you adjust that? With some user backends like LDAP the user id could just be a UUID